zkevm-circuits/src/super_circuit.rs

```diff
@@ -337,15 +303,16 @@ mod super_circuit_tests {
         let k = k + 1;
```

Comment:
Note: I added this to get a working `k` but this is very ugly :crying_cat_face: 
This is required because I didn't add an estimation of `k` for the TxCircuit.
Ideally we should have an estimate of the minimum k for each circuit given the inputs, and then pick the max one for the SuperCircuit.

Reply:
Should we file an issue for that in all of our circuits???

Reply:
Yes! I've just opened the issue: https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/690